docs/articles/documentation/test-api/selecting-page-elements/selectors/using-selectors.md

```diff
@@ -184,4 +185,15 @@ Within the selector timeout, the selector is executed over and over again, until
 DOM node or the timeout exceeds. If TestCafe cannot find the corresponding node in the DOM, the test fails.
 
 > Note that you can require that the node returned by the selector is visible.
-To do this, use the [visibilityCheck](selector-options.md#optionsvisibilitycheck) option.
\ No newline at end of file
+To do this, use the [visibilityCheck](selector-options.md#optionsvisibilitycheck) option.
+
+## Debug Selectors
+
+TestCafe outputs information about failed selectors to test run reports.
+
+When you try to use a selector that does not match any DOM element, the test fails and an error is thrown.
+
+The error message indicates which selector has failed. If a fail occurs in a [selector methods chain](functional-style-selectors.md),
```

Comment:
I would remove this link.
If I clicks a `selector methods chain` link, then I expect that topic about `selector methods chain` will be opened. The `method chaining' is not a main idea of `functional-style-selectors.md` topic.